src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -159,6 +163,24 @@ const getAuthenticatorDescription = (
   return undefined;
 };
 
+const getUsageDescription = (option: IdxOption): string | undefined => {
+  // @ts-ignore IdxAuthenticator missing allowedFor property
+  const { allowedFor } = option.relatesTo;
+  if (!allowedFor) {
+    return undefined;
+  }
```

Comment:
i don't think this guard is necessary since the `switch` case will fall through to `default` which is the same